CHANGELOG.md

```diff
@@ -0,0 +1,515 @@
+# Changelog
+
+All notable changes are documented in this file.
+The sections should follow the order `Packaging`, `Added`, `Changed`, `Fixed` and `Removed`.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## [Unreleased]
+
+## [5.0.0] - 2023-04-13
```

Comment:
I honestly don't really like the brackets around the version, they just seem very noisy to me.

Isn't `## 5.0.0 - 2023-04-13` just as good, if not better? I know that you likely just followed `keepachangelog.com`, but I'm personally not a massive fan of this (it also makes interacting with the text in an editor more difficult imo).

Reply:
The brackets are for markdown links (defined at the bottom of the file).

Reply:
Exactly what @maxrake said. It makes them links [when rendered](https://github.com/phylum-dev/cli/blob/changelog/CHANGELOG.md).

In fact, if you go far enough down in the file, some early versions don't have these links, because I didn't have enough information to include them (and didn't feel it was worth manually looking each one up).